build.gradle.kts

```diff
@@ -10,25 +10,6 @@ configurations {
   testImplementation { exclude(group = "org.junit.vintage") }
 }
 
```

Comment:
aligning to the template project - tests are run in one stage now and int tests can be run as JUnit tests now